app/models/ark_minter.rb

```diff
@@ -74,6 +75,16 @@ def instance
 
   private
 
+  def synchronize_states!(current_inst)
+    other_inst = MinterState.where.not(namespace: current_inst.namespace).first
```

Comment:
Should we make this an `each` rather than just selecting the `first` result?

If at some point we had more than two namespaces it might save some time down the road?